### ISSUE TITLE
Stuck at No Music Found screen

### ISSUE THREAD

**Issue by Mhowser** [enricocid/Music-Player-GO#281]
![Screenshot_20210311-023452](https://user-images.githubusercontent.com/24195572/110774847-18faa100-8213-11eb-85d9-a00a1a91e2b6.png)
I have music on my device, which is a Motorola Moto e 2020 android 10

**Comment by enricocid** [comment]
Hi, where is the music stored?
It means Mediastore database has no entries. You can try to reboot the
phone or move the music to another location👍🏻

There's no much I can do about this unfortunately, it depends by the system
and Mediastore

Il gio 11 mar 2021, 11:41 Mhowser ***@***.***> ha scritto:

**Comment by Mhowser** [comment]
The music files are in `/storage/emulated/0/Music`

**Comment by JayUK** [comment]
I have the same issue :(

My music is located in /sdcard/Music

Any ideas?

**Comment by enricocid** [comment]
Could you try with phonograph and let me know if you experience the same issue?

**Comment by JayUK** [comment]
Hi there,

Please ignore me.....

 I did an uninstall/reboot/reinstall. It asked for permissions to my storage and now it's working *weird*

**Comment by enricocid** [comment]
Yeah, same here.
MediaStore/Android is weird.

Gonna implement a new method to query the music to prevent this issue😉

Closing ...

**Comment by Mhowser** [comment]
Is it possible to use VLC's method for scanning music?

**Comment by aberfan** [comment]
I have the same issue, and I tried removing and reinstalling.  Why isn't there some sort of option to select? The app just exiting is just bizarre.

**Comment by sam-github** [comment]
Sorry, I may be having a similar issue. I'm using Synology Drive to mark files on my NAS for "Offline Access", and was hoping that would be sufficient for the Music Player to find and recognize the music, but I'm not seeing anything. I did a restart of the device to see if that would help. Fwiw, I turned wifi off, and I can play the songs using the Synology Drive app.

What exactly are the requirements for audio files on my android device to be found and playable by the Music Player?
